.ahoy.yml

```diff
@@ -121,8 +119,40 @@ commands:
   test-bdd:
     usage: Run BDD tests.
     cmd: |
+      ahoy start-ckan-job-workers &
+      sleep 5
       ahoy cli "behave ${*:-/app/test/features}" || \
       [ "${ALLOW_BDD_FAIL:-0}" -eq 1 ]
+      ahoy stop-ckan-job-workers
+
+  start-ckan-job-workers:
+    usage: Starts CKAN background job workers
+    cmd: |
+      ahoy title 'Starting CKAN background job workers'
+      ahoy start-ckan-job-worker-priority &
+      ahoy start-ckan-job-worker-bulk &
+
+  start-ckan-job-worker-priority:
+    usage: Starts CKAN background job worker
+    cmd: |
+      ahoy title 'Starting CKAN background job worker priority'
+      ahoy cli "cd /usr/lib/ckan/default/src/ckan && \
+                paster jobs clear priority --config=/app/ckan/default/production.ini && \
```

Comment:
Why clear the queue?

Reply:
most likely to reset testing.

Reply:
@ThrawnCA It is really just there to be safe for local dev testing when sometimes the worker jobs were still running and did not stop successfully giving me peculiar results. 
In circleci this is not an issue as the containers are created before and destroyed after the tests complete but on local dev/testing this is not always the case.
I think its safer and cleaner with this approach for local BDD dev/testing and no harm with leaving in.